x-pack/plugins/monitoring/public/components/beats/overview/overview.js

```diff
@@ -30,46 +30,40 @@ function renderLatestActive(latestActive, latestTypes, latestVersions) {
     return (
       <EuiFlexGroup wrap>
         <EuiFlexItem>
-          <EuiPanel>
-            <EuiTitle size="s">
-              <h2>
-                <FormattedMessage
-                  id="xpack.monitoring.beats.overview.activeBeatsInLastDayTitle"
-                  defaultMessage="Active Beats in Last Day"
-                />
-              </h2>
-            </EuiTitle>
-            <EuiSpacer size="s" />
-            <LatestActive latestActive={latestActive} />
-          </EuiPanel>
+          <EuiTitle size="s">
+            <h2>
+              <FormattedMessage
+                id="xpack.monitoring.beats.overview.activeBeatsInLastDayTitle"
+                defaultMessage="Active Beats in Last Day"
```

Comment:
```suggestion
                defaultMessage="Active Beats in last day"
```

Please use sentence case everywhere.
Also, why is the timeframe for these last day and not dependent on the time selector?

Reply:
I honestly don't know, but that's a good point.

Reply:
FWIW, it is accurate. I'd suggest leaving it for now, as it's not a bug. I wonder if the beats team had requested this.